pages/countries/index.js

```diff
@@ -20,24 +25,55 @@ export default function Countries({ surfspots }) {
     router.push(`/`);
   }
 
+  function handleChange(event) {
+    setSearchTerm(event.target.value);
+    let searchValue = event.target.value;
```

Comment:
ist das hier dann nicht schon der State "searchTerm", wenn du in der Zeile drüber das so setzt? und müsste es nicht `event.target.(elements.)place.value` sein? ich habs in deiner Preview mal ausprobiert und es funktioniert prima, während ich was eingebe, aber sobald ich auf die Lupe klicke, ist alles weg...